src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -1949,9 +1949,9 @@ static void M(A a)
                 // (8,9): error CS1656: Cannot assign to 'E' because it is a 'method group'
                 //         a.E! += a.E!; // 1
                 Diagnostic(ErrorCode.ERR_AssgReadonlyLocalCause, "a.E").WithArguments("E", "method group").WithLocation(8, 9),
-                // (9,13): error CS0019: Operator '!=' cannot be applied to operands of type 'method group' and '<null>'
+                // (9,13): error CS8652: The feature 'inferred delegate type' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
                 //         if (a.E! != null) // 2
-                Diagnostic(ErrorCode.ERR_BadBinaryOps, "a.E! != null").WithArguments("!=", "method group", "<null>").WithLocation(9, 13),
+                Diagnostic(ErrorCode.ERR_FeatureInPreview, "a.E").WithArguments("inferred delegate type").WithLocation(9, 13),
```

Comment:
Please test with LangVer=preview (I assume the delegate is considered not-null, so warnings on lines 2 and 9 go away)

Reply:
`DelegateTypeTests` contains several specific tests for language version. For other tests, I chose to either use `/langversion:9` or `/langversion:preview` but not both since both seemed unnecessary. Let me know if you'd prefer the other tests to use `/langversion:preview`.